setup.py

```diff
@@ -14,6 +15,27 @@
 
 from setuptools import find_packages
 from setuptools import setup
+from setuptools.command.test import test as TestCommand
+
+
+class PyTest(TestCommand):
```

Comment:
This is used for "./setup.py test", invoked by travis and coverage.
Removing it breaks everything